src/Common/MetricHelper.cs

```diff
@@ -508,7 +508,7 @@ private void PopulateConfigMetricsFromQos(AzurePSQoSEvent qos, IDictionary<strin
             {
                 foreach (var configMetric in qos.ConfigMetrics)
                 {
-                    eventProperties[configMetric.ConfigKey] = configMetric.ConfigValue;
+                    eventProperties[configMetric.Value.TelemetryKey] = configMetric.Value.ConfigValue;
```

Comment:
I think using the real config key as the key of eventProperties will simplify the code in Accounts. But you can decide